src/main/kotlin/no/nav/klage/oppgave/clients/OppgaveClient.kt

```diff
@@ -120,6 +122,38 @@ class OppgaveClient(
         //TODO
         return TEMA_SYK
     }
+
+    fun endreHjemmel(oppgaveId: Int, hjemmel: String): Oppgave {
+        var oppgave = oppgaveWebClient.get()
+            .uri { uriBuilder ->
+                uriBuilder.pathSegment("{id}}").build(oppgaveId)
+            }
+            .header("Authorization", "Bearer ${stsClient.oidcToken()}")
+            .header("X-Correlation-ID", tracer.currentSpan().context().traceIdString())
+            .header("Nav-Consumer-Id", applicationName)
+            .retrieve()
+            .bodyToMono<EndreOppgave>()
+            .block() ?: throw RuntimeException("Oppgave could not be fetched")
+        logger.info("Endrer hjemmel for oppgave {} fra {} til {}", oppgave.id, oppgave.metadata?.get(HJEMMEL), hjemmel)
+        if (oppgave.metadata == null) {
+            oppgave.metadata = mutableMapOf()
+        }
+        oppgave.metadata!![HJEMMEL] = hjemmel
```

Comment:
Litt kjipt at Kotlin ikke kan utlede at metadata ikke er null. Men men.

Reply:
Ja, jeg ville trodd det, men den mumla noe om at et-eller-annet var for komplekst, så det gikk ikke..